custom/icds/tasks.py

```diff
@@ -111,6 +111,8 @@ def run_weekly_indicators():
 
         for user_id in generate_user_ids_from_primary_location_ids(domain, get_awc_location_ids(domain)):
             language_code = get_language_code(user_id, telugu_user_ids, marathi_user_ids)
+            if language_code != TELUGU:
```

Comment:
yeah only Andhra Pradesh will use Telugu so this is fine for now (we derive Telugu language code by checking state location), and based on the rollout plan it seemed like the next step will be the Hindi speaking states so we can probably just keep this simple and use language code as the way to do our rollout